### PR TITLE
[#7661] adding docs about rate limiting

### PR DESCRIPTION
**Pull request by NikolayRys**

### CODE REVIEW

**Comment by NikolayRys** [comment]
Please review: https://gresb.github.io/api-docs/#batch-asset-operations

File: source/includes/_batch-operations.md [hunk 1]
```diff
@@ -104,6 +104,25 @@ and `always_update` to bypass the validation procedure.
...
+
+These values are experimental and therefore are subject to change.
+However, you are unlikely to reach them during normal operation.
+
+The information about the throttling is also provided in the following headers:
+
+  - `X-RateLimit-Limit` contains the maximum number of allowed requests during a time period.
+  - `X-RateLimit-Reset` expected counter reset time in UTC epoch seconds.
+
+If a response is over the limit, it's rejected with 429 status and an additional header is present:
```
Comment (joeyhorst) [inline source/includes/_batch-operations.md]: Is the header only present when a response is over the limit or is it always present?

Comment (NikolayRys) [reply source/includes/_batch-operations.md]: The former - only when over the limit. 
It's a bit tough to calculate it for the exact value. It's possible, but would require some additional points invested, which means that so far we know only when they are exhausted.

Comment (teoulas) [reply source/includes/_batch-operations.md]: Well it was part of the spec to communicate the current usage / remaining requests. I also asked you in the [pull request](https://github.com/GRESB/gresb-reports/pull/7870#discussion_r295733163) if you tried a gem that seems to do it, but you didn't answer? Was it broken?

Comment (NikolayRys) [reply source/includes/_batch-operations.md]: I have answered there. It might not be broken now, but will break eventually because it's not maintained.

**Comment by NikolayRys** [comment]
Added version information.

**Pull request merged**